xcube/core/tilingscheme.py

```diff
@@ -36,11 +36,11 @@
     'EPSG:3857',
     'urn:ogc:def:crs:OGC:1.3:EPSG::3857',
 )
-GEOGRAPHIC_CRS_NAME = 'CRS84'
+GEOGRAPHIC_CRS_NAME = 'OGC:CRS84'
 GEOGRAPHIC_CRS_ALIASES = (
     'EPSG:4326',
     'urn:ogc:def:crs:OGC:1.3:EPSG::4326',
-    'CRS84',
+    'OGC:CRS84',
```

Comment:
```suggestion
    'OGC:CRS84',
    'CRS84',
```

This is a list of aliases recognized by the class, so we should keep the old identifier as well as adding the new one.